app/utils/markdown/index.ts

```diff
@@ -275,7 +275,7 @@ export const getMarkdownImageSize = (
 
 export const computeTextStyle = (textStyles: MarkdownTextStyles, baseStyle: StyleProp<TextStyle>, context: string[]) => {
     const contextStyles: TextStyle[] = context.map((type) => textStyles[type]).filter((f) => f !== undefined);
-    return contextStyles.length ? concatStyles(baseStyle, contextStyles) : baseStyle;
```

Comment:
Why not using concatStyles anymore? And if so, do we need concatStyles at all?

Reply:
Not sure why we added to begin with. A `StyleProp` supports nested lists (`[[[[{color: 'blue'}]]]]` is a valid style prop), and React Native takes care of the style flattening, so there is no need to be fancy here.

And yes, I think `concatStyles` can be removed. Will do.